### ISSUE TITLE
'Glue' object has no attribute 'get_statements when interrupt dbt run

### ISSUE THREAD

**Issue by zhangyuan** [aws-samples/dbt-glue#203]
### Describe the bug

When dbt command is running, hit `ctrl+c` to interupt it. It raise the error

```
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
....
    raise AttributeError(
AttributeError: 'Glue' object has no attribute 'get_statements'
```

(the exceptions is raised from `cancel` method)

### Steps To Reproduce

Run `dbt run` command, and hit `ctrl+c`. The exception is printed to the terminal.

### System information

```
 dbt --version
Core:
  - installed: 1.5.2
  - latest:    1.6.0 - Update available!

  Your version of dbt-core is out of date!
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation

Plugins:
  - glue:  1.5.3 - Update available!
  - spark: 1.5.0 - Update available!

  At least one plugin is out of date or incompatible with dbt-core.
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation
```

**The operating system you're using:**

MacOS

**The output of `python --version`:**

```
python --version
Python 3.9.6
```
### Additional context

I checked the boto3 API, there's no method called `get_statements`, but a `get_statement` ( https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue/client/get_statement.html#Glue.Client.get_statement ). I guess the API name changed?

**Comment by AmineIzanami** [comment]
@zhangyuan indeed that seems like a bug, thanks for raising it, we will add it on our fix list for next patch.

**Comment by zhangyuan** [comment]
@AmineIzanami thank you. If so, I will not raise the PR for it..

**Comment by AmineIzanami** [comment]
Fixed in #206
